06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * The Inverted Index class.
+ * @class
+ */
+class InvertedIndex {
+
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.unique = [];
+    this.tableObj = {};
+  }
+
+  /**
+   * A method that tokenizes the string that is passed through it
+   * @param  {string} data the string  from the "text" key in the JSON
+   * @return {Array} containing String.
+   */
+  tokenize(data) {
+    this.terms = data.replace(/[^\w\s]/gi, ' ')
+      .match(/\w+/g);
+    return this.terms;
+  }
+
+  /**
+   * A method that filters the array passed into it for unique words
+   * @param {String} data Array of strings
+   * @returns {String} Returns array of unique words
+   */
+  uniqueWords(data) {
```

Comment:
Thank you for pointing this out, I have seen the error in the naming convention, and have rectified this issue.